python/vyos/template.py

```diff
@@ -674,6 +674,26 @@ def add_includes(name):
         add_includes(name)
     return out_list
 
+@register_filter('nft_accept_invalid')
+def nft_accept_invalid(ether_type):
+    ether_type_mapping = {
+        'dhcp': 'udp sport 67 udp dport 68',
```

Comment:
If one passes in an invalid `ether_type` and exception would be raised. We should return a proper RuntimeError with a meaniungful error message like https://github.com/vyos/vyos-1x/blob/80652326e6e5f9d193ead77e0bee62f981e79301/python/vyos/template.py#L1109-L1111